.github/tailor.yaml

```diff
@@ -2,9 +2,9 @@ rules:
   - name: commit title
     description: all commit titles must have a scope, be lowercase, and be no more than 50 characters
     expression:  |-
-      .commits all((.message test "^([a-z/]+)|(\\*): [a-z \\-'.]+") and (.message test "^.{0,50}"))
+      .commits all((.message test "^(Revert \")?([a-z/]+|\\*): [a-z0-9 \\-'\"./]+") and (.message test "^.{0,50}"))
```

Comment:
I think the Revert check should be used as an entire optional block where the commit message length is extended to include `Revert \"` & `\"`, otherwise won't this fail if we do a revert on a commit that already had a 41+ character title?

Reply:
Hmmmmmmmmmmmmmmmmmmmmmmmmm. Yeah, let's rework that a bit.